Makefile

```diff
@@ -304,7 +304,8 @@ endif
 bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
 	$(OPERATOR_SDK) generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
-	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS)
+	cd config/manager && $(KUSTOMIZE) edit set image daemonset=$(IMG_DMST)
+	$(KUSTOMIZE) build config/manifests | sed -e "s|<IMG>|$(IMG)|g" -e "s|<IMG_DMST>|$(IMG_DMST)|g" | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS)
```

Comment:
Is there a problem if we replace <IMG> with ${IMG} directly inside the yaml instead of replacing it here?

Reply:
Okay, What I meant is to remove the replace step using `sed` by placing the `$IMG` inside the yaml file instead of  `<IMG>`